### PR TITLE
Update Max base image

### PR DESCRIPTION
**Pull request by feihugis**
This PR updates the Max base image and change the API title and description.

### CODE REVIEW

File: api/predict.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    @MAX_API.marshal_with(predict_response)
+    def post(self):
+        """Make a prediction given input data"""
+        result = {'status': 'error'}
+
+        args = input_parser.parse_args()
+        try:
+            input_data = args['image'].read()
+            image = self.model_wrapper._read_image(input_data)
+        except OSError as e:
```
Comment (bdwyer2) [inline api/predict.py]: Maybe this should be `IOError` instead of `OSError`?

File: config.py [hunk 2]
```diff
@@ -7,23 +7,15 @@
 # Application settings
 
 # API metadata
-API_TITLE = 'Model Asset Exchange Server'
-API_DESC = 'An API for serving models'
+API_TITLE = 'MAX Facial Recognizer'
+API_DESC = 'Detect faces from the image and generate embeddings for the detected faces'
```
Comment (bdwyer2) [inline config.py]:
```suggestion
API_DESC = 'Recognize faces in an image and extract embedding vectors for each face.'
```
per the description here https://developer.ibm.com/exchanges/models/all/max-facial-recognizer/

File: tests/test.py [hunk 3]
```diff
@@ -35,7 +36,7 @@ def test_predict():
     model_endpoint = 'http://localhost:5000/model/predict'
 
     # Test by the image with multiple faces
-    img1_path = 'assets/codait.jpeg'
+    img1_path = 'assets/Bryan.png'
```
Comment (BryanCutler) [inline tests/test.py]: Is this test data provided with an appropriate Apache 2.0 license?

File: api/predict.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    @MAX_API.expect(input_parser)
+    @MAX_API.marshal_with(predict_response)
+    def post(self):
+        """Make a prediction given input data"""
+        result = {'status': 'error'}
+
+        args = input_parser.parse_args()
+        try:
+            input_data = args['image'].read()
+            image = self.model_wrapper._read_image(input_data)
```
Comment (djalova) [inline api/predict.py]: I think _read_image() should be called in _pre_process()

File: tests/test.py (hunk 3 above)
Comment (feihugis) [reply tests/test.py]: @BryanCutler Really sorry about this! I will delete the image.

Comment (feihugis) [reply tests/test.py]: My bad! I should ask you before using your photo. Would you mind if I use your photo in this repo?

**Pull request merged**